packages/test-vectors/src/generators/LEIevidenceDocument.js

```diff
@@ -0,0 +1,93 @@
+const faker = require('faker');
+
+const getLEIevidenceDocument = () => {
+  const example = {
+    '@context': ['https://w3id.org/traceability/v1'],
+    type: 'LEIevidenceDocument',
+    "data": {
+      "type": faker.lorem.word(),
+      "id": faker.random.uuid(),
+      "attributes": {
+        "lei": faker.lorem.word(),
+        "entity": {
+          "legalName":  {
+            "name": faker.company.companyName(),
+            "language": null
+          },
+          "otherNames": [],
+          "transliteratedOtherNames": [],
+          "legalAddress":  {
+            "language":  null,
+            "addressLines": [],
+            "addressNumber": null,
+            "addressNumberWithinBuilding": null,
+            "mailRouting": null,
+            "city": faker.address.city(),
+            "region": faker.address.county(),
+            "country":  faker.address.country(),
+            "postalCode": faker.address.zipCode()
+          },
+          "headquartersAddress": {
+            "language": null,
+            "addressLines": [],
+            "addressNumber": null,
+            "addressNumberWithinBuilding": null,
+            "mailRouting": null,
+            "city": faker.address.city(),
+            "region": faker.address.county(),
+            "country":  faker.address.country(),
+            "postalCode": faker.address.zipCode()
+          },
+          "registeredAt": {
+            "id": faker.random.uuid(),
+            "other": null
+          },
+          "registeredAs": faker.address.city(),
+          "jurisdiction":  faker.lorem.word(),
+          "category": null,
+          "legalForm": {
+            "id": faker.random.uuid(),
+            "other": null
+          },
+          "associatedEntity": {
+            "lei": null,
+            "name": null
+          },
+          "status": faker.lorem.word(),
```

Comment:
maybe better to make this an enum.

Reply:
"VALID", "INVALID", etc...

Reply:
see https://github.com/w3c-ccg/traceability-vocab/blob/main/packages/test-vectors/src/generators/MechanicalProperty.js#L6

Reply:
status field randomly will be selected from the array with predefined statuses